src/pages/zh/about.js

```diff
@@ -0,0 +1,244 @@
+import '../../styles/pages/zh/about.scss'
+import '../../lib/graphql/image'
+
+import React from 'react'
+import { graphql, Link } from 'gatsby'
+import Img from 'gatsby-image'
+import { Button } from '@seagreenio/react-bulma'
+import Particles from 'react-particles-js'
+
+import Layout from '../../components/layout'
+import SEO from '../../components/seo'
+import ScrollTopButton from '../../components/scrollTopButton'
+import { workEnv } from '../../data/zh/career-work-env'
+import { offices, contacts } from '../../data/zh/contact-us'
+import { mapData } from '../../data/zh/map'
+
+const About = ({ data }) => {
+  const { aboutHeroSVG, timeLinePNG, mapSVG } = data
+  const particleConfig = {
+    particles: {
+      color: {
+        value: '#000000',
+      },
+      line_linked: {
+        color: {
+          value: '#000000',
+        },
+      },
+      number: {
+        value: 50,
+      },
+      size: {
+        value: 3,
+      },
+    },
+  }
+  const introVideoConfig = {
+    src: 'https://download.pingcap.com/videos/pingcap-intro-v2.mp4',
+    type: 'video/mp4',
+    poster: 'https://download.pingcap.com/images/video-poster.jpg',
+    controls: true,
+    playsInline: true,
+  }
+  return (
+    <Layout>
+      <SEO
+        title="About PingCAP"
+        description="Story about PingCAP, the team behind TiDB"
+      />
+      <article className="PingCAP-About-ZH">
+        <section className="hero is-medium">
+          <div className="hero-body">
+            <div className="container has-text-centered">
+              <img src={aboutHeroSVG.publicURL} alt="Community Hero" />
+            </div>
+          </div>
+        </section>
+
+        <section className="section intro-section">
+          <div className="container">
+            <h1 className="title">关于我们</h1>
+            <p className="paragraph">
+              PingCAP是一家开源的新型分布式数据库公司，秉承开源是基础软件的未来这一理念，
+              PingCAP持续扩大社区影响力，致力于前沿技术领域的创新实现。其研发的分布式关系型数据库
+              TiDB项目，具备「分布式强一致性事务、在线弹性水平扩展、故障自恢复的高可用、
+              跨数据中心多活」等核心特性，是大数据时代理想的数据库集群和云数据库解决方案。
+              目前，已被近1000家不同行业的领先企业应用在实际生产环境，涉及互联网、游戏、
+              银行、保险、证券、航空、制造业、电信、新零售、政府等多个行业，包括美国、欧洲、
+              日本、东南亚等海外用户。
+            </p>
+            <Button
```

Comment:
These buttons do not match (by interaction) the other buttons in the whole site. Is this on purpose?

Reply:
yes, and i think they are consistent with previous chinese website by interaction

Reply:
> These buttons do not match (by interaction) the other buttons in the whole site. Is this on purpose?

yes, and i think they are consistent with previous chinese website by interaction